frmts/netcdf/netcdfdataset.cpp

```diff
@@ -2656,6 +2656,16 @@ void netCDFDataset::SetProjectionFromVar( int nGroupId, int nVarId,
                 }
                 if( pszWKT )
                 {
+                    bGotGdalSRS = true;
+                    CPLDebug("GDAL_netCDF", "setting WKT from GDAL");
```

Comment:
I added a `bGotCfWktSRS` to differentiate.

Reply:
I had thought about that and was thinking that since GDAL is planning to drop `spatial_ref` and only export `crs_wkt` it may be a GDAL generated attribute in the future. :man_shrugging: